packages/lesswrong/components/tagging/TagDiscussionButton.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { Components, registerComponent } from '../../lib/vulcan-lib';
+import { Link } from "../../lib/reactRouterWrapper";
+import CommentOutlinedIcon from "@material-ui/icons/ModeCommentOutlined";
+import { useHover } from "../common/withHover";
+
+const styles = (theme: ThemeType): JssStyles => ({
+  discussionButton: {
+    ...theme.typography.commentStyle,
+    ...theme.typography.body2,
+    ...theme.typography.uiStyle,
+    color: theme.palette.grey[700],
+    display: "flex",
+    '& svg': {
```

Comment:
This should never be necessary, as mentioned in the other comment. Apply a css class directly to the `CommentOutlinedIcon` component.